Dockerfile

```diff
@@ -1,29 +1,30 @@
-FROM golang:1.13 as secrets-provider-builder
+FROM golang:1.13 as secrets-provider-builder-base
 MAINTAINER CyberArk Software Ltd.
 
 ENV GOOS=linux \
     GOARCH=amd64 \
     CGO_ENABLED=0
 
-COPY . /opt/secrets-provider-for-k8s
+RUN apt-get update
+
+RUN go get -u github.com/jstemmer/go-junit-report && \
+    go get github.com/smartystreets/goconvey
 
 WORKDIR /opt/secrets-provider-for-k8s
 
 EXPOSE 8080
 
-RUN apt-get update && apt-get install -y jq
```

Comment:
is `jq` not required anymore?

Reply:
Since when it is required for building go?

Reply:
Also, I don't know why we need the busybox tools in production...

Reply:
Also, we can use `FROM scratch` in the main containers instead of `FROM alpine`

Reply:
let's do it one step at a time. Once we finish this PR we can challenge the base image (`scratch/alpine`) and the `busybox` tools.

They may be a leftover but this PR is already big enough to chew on :)

Reply:
@abrahamko can you create an issue for challenging the base image (scratch/alpine) and the busybox tools?